src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -4,11 +4,6 @@ createTexture validation tests.
 TODO: review existing tests and merge with this plan:
 > All x= every texture format
 >
-> - usage flags
->     - {0, ... each single usage flag}
->     - x= every texture format
-> - every possible pair of usage flags
->     - with one common texture format
 > - any other conditions from the spec
 > - ...?
```

Comment:
It looks like this TODO can be removed? or at least reduced simply to "TODO: check the spec for any other validation conditions that need to be tested here" (we should do this TODO very soon, since the file is getting fairly complete, so we can call the file "Done").

Reply:
How about to have a separate patch to clean createTexture? I noticed a few things like some comments, some change in capability_info.ts, and maybe a few more tests for corner cases.

Reply:
SGTM